src/lib/db/schema/invitations.ts

```diff
@@ -0,0 +1,23 @@
+import { json, pgTable, text, timestamp } from "drizzle-orm/pg-core";
+import { users } from "~/lib/db/schema/users";
+
+export const invitations = pgTable("invitation", {
+  id: text("id").primaryKey(),
+  userId: text("user_id").references(() => users.id),
+  customFields: json("custom_fields"),
```

Comment:
[질문]
json 타입은 `notNull` 안해도 괜찮나요?

Reply:
@9keyyyy 
타입스크립트 호환성을 위해 `$type`에 타입을 추가해주세요!
https://orm.drizzle.team/docs/column-types/pg#json
```suggestion
  customFields: json("custom_fields").notNull().$type<Record<string, any>>(),
```

Reply:
반영했습니당 ㅎㅎ 확인 감사합니다~!